proc_dash/utility.py

```diff
@@ -0,0 +1,175 @@
+import base64
+import io
+import json
+from pathlib import Path
+from typing import Optional, Tuple
+
+import pandas as pd
+
+SCHEMAS_PATH = Path(__file__).absolute().parents[1] / "schemas"
+
+
+def get_required_bagel_columns() -> list:
```

Comment:
This file is a good first step in separating out functionality from the app script itself – and I'm guessing as dash grows you'll likely even split up this file – or maybe even consider converting this into something more object oriented that handles the bagel i/o.

One suggestion I do have for this file is either sorting the functions by functionality or alphabetizing all of the functions.